packages/devtools_app/lib/src/logging/flutter/logging_screen.dart

```diff
@@ -357,6 +362,33 @@ class _MessageColumn extends LogMessageColumn
           ),
         ],
       );
+    } else if (data.kind == 'stdout') {
+      // TODO(helin24): Move this to logging controller when dart:html is removed.
+      const Color color = Color.fromARGB(0xff, 0x00, 0x91, 0xea);
+
+      final RichText richText = RichText(
+        text: TextSpan(
+          children: maybeConvertToAnsiText(
+              // TODO(helin24): Recompute summary length considering ansi codes?
```

Comment:
Good todo. You can remove the question mark as this a valid case to optimize but not critical to optimize now.